rs/ledger_suite/tests/sm-tests/src/lib.rs

```diff
@@ -3028,7 +3094,10 @@ pub fn test_migration_resumes_from_frozen<T>(
         .unwrap();
 
     const APPROVE_AMOUNT: u64 = 150_000;
+    const TRANSFER_AMOUNT: u64 = 100;
```

Comment:
Should we maybe vary the transfer amount to test that the migration doesn't assign the same amount to all accounts?

Reply:
Good point! I changed to `TRANSFER_AMOUNT + i` for the i-th recipient.